build.sh

```diff
@@ -36,8 +36,7 @@ done
 # Make sure we have the tools we need installed
 sudo apt-get -q install $devtools -y --no-install-recommends
 sudo apt-get update
-sudo apt-get install debhelper
-sudo apt-get install dpkg-dev
+sudo apt-get install debhelper dpkg-dev
```

Comment:
Are you pushing commas around? ;) This commit adds or removes zero functionality.